scripts/do-microblaze-configure

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
should be named do-microblazeel-configure to allow for a future big-endian version

Reply:
Done.